src/pyscipopt/scip.pxi

```diff
@@ -3908,11 +6472,21 @@ cdef class Model:
         return mappedvar
 
     def getBendersAuxiliaryVar(self, probnumber, Benders benders = None):
-        """Returns the auxiliary variable that is associated with the input problem number
+        """
+        Returns the auxiliary variable that is associated with the input problem number
+
+        Parameters
+        ----------
+        probnumber : int
+            the problem number for which the target variable belongs, -1 for master problem
+            (default=-1)
```

Comment:
why default -1?

Reply:
True. This is probably outdated. I will change it.